backend/src/main/java/com/benine/backend/video/StreamNotAvailableException.java

```diff
@@ -0,0 +1,13 @@
+package com.benine.backend.video;
+
+/**
+ * Defines an exception thrown when a stream is not available.
+ */
+public class StreamNotAvailableException extends Exception {
+
+  /**
+   * Should be thrown if a stream is not available.
+   */
+  public StreamNotAvailableException() {}
```

Comment:
Maybe you could provide some info about the Stream like, the stream for what camera or what link and maybe even a reason why its not available for example.